CHANGELOG.md

```diff
@@ -8,6 +8,38 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [1.1.0] - 2018-09-05
+
+### Added
+- New `WordPress.PHP.NoSilencedErrors` sniff. This sniff replaces the `Generic.PHP.NoSilencedErrors` sniff which was previously used and included in the `WordPress-Core` ruleset.
+    The WordPress specific version of the sniff differs from the PHPCS version in that it:
```

Comment:
The raw Markdown file is showing several indented lines (no bullets), but the preview just shows these run on as part of the previous sentence (no line breaks).

Reply:
Correct. This is one of the GH markdown intricacies.
I could add a new line between these lines, which will in turn make it an (indented) proper new line, however, if I do, that changes the padding/margin around every line in the bullet list, which would make the mark-up inconsistent with the rest of the document and looks weird anyway.

Reply:
The weird thing is that the markdown gets parsed differently in a GH comment vs a markdown document on GH. In markdown comments, all works as expected, it's just markdown documents which behave strangely.